### DISCUSSION TITLE
Resetting the video counter w/o full factory reset

### DISCUSSION THREAD

**Discussion by IcedQuinn** [gopro/labs#469] [Q&A]
Hello, I have a Hero 9 Black and the support page says resetting the counter is only available with the Labs firmware. I have installed this but cannot find what you actually have to do to reset the numbers. https://community.gopro.com/s/article/Is-it-Possible-to-Reset-the-Camera-File-Counter?language=en_US

Using the tool to apply a base name for the camera--and setting the base name to nothing--doesn't seem to do it. The camera acknowledges the name has been reset to default (with the text overlay upon scanning the code) but when I tested on a card it still had a recording counter from all previous sessions.

**Comment by dnewman-gpsw** [comment]
I'm not aware what the command would have been (if ever), I think that link is in error.  It is on the back log to add this feature.
